TESTS.md

```diff
@@ -0,0 +1,38 @@
+### [<= Back to `README.md`](./README.md)
+## Running Unit Tests
+### Create `token.txt` in `~/Smart-Ledger/Back-End/tests/`
+```shell
+touch tests/token.txt
+```
+
+
+#### In `Smart-Ledger/Front-End/.env.development`, change endpoint:
+```shell
+//.env.development
+REACT_APP_API_ENDPOINT=http://ddns.absolutzero.org:5555/
```

Comment:
This step is not necessary as that specific endpoint URL is temporary (basically it's just the back-end spun up on my own PC to share with y'all).
You can simply spin up the back-end (API) container locally: that way the API endpoint is just localhost:5000 like it is set by default

Reply:
add